source/index.html.md

```diff
@@ -540,26 +554,119 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
+    <span class="type">(<code>storing</code>|<code>using</code>)</span>
```

Comment:
Two points:

1. We use `text_on_statement` to avoid the TLA `tos`; likewise, we should just use `credential_on_file`.
2. It is hard not to like the simplicity/brevity of `store|use` (courtesy @mt-clearhaus, https://github.com/clearhaus/gateway-api-docs/pull/111/files#r500507071).

Thus:

```suggestion
  <dt>credential_on_file
    <span class="type">(<code>store</code>|<code>use</code>)</span>
```

In return I have marked https://github.com/clearhaus/gateway-api-docs/pull/111/files#r497447364 and https://github.com/clearhaus/gateway-api-docs/pull/111/files#r497425813 as resolved.